### PR TITLE
Updates codepens using Can 5 code

### PR DESCRIPTION
**Pull request by m-mujica**
Counter: https://codepen.io/bitovi/pen/bGGNgyx
Progressive Loading: https://codepen.io/bitovi/pen/bGGNqbm
CRUD model: https://codepen.io/bitovi/pen/vYYEmPE

### CODE REVIEW

**Comment by m-mujica** [comment]
btw, I went ahead and merged / released https://github.com/bitovi-components/bit-hello-world, and  https://github.com/bitovi-components/bit-goodbye-moon

File: docs/can-guides/setup/setting-up-canjs.md [hunk 1]
```diff
@@ -933,19 +933,19 @@ With CanJS downloaded or installed, use it to create an `index.html` page with a
...
-        The server says: {{messagePromise.value}}
-    `,
-	ViewModel: {
-		feels: { default: "😍" },
-        messagePromise: {
-            default: () => can.ajax({url: "/message"})
-        }
-	}
-});
+class MyApp extends can.StacheElement {
```
Comment (phillipskevin) [inline docs/can-guides/setup/setting-up-canjs.md]: Forgot to look at the other changes before approving. Does `can.StacheElement` work?

Comment (m-mujica) [reply docs/can-guides/setup/setting-up-canjs.md]: I'll try. I actually wanted to tag you on this since I wasn't sure.

Comment (m-mujica) [reply docs/can-guides/setup/setting-up-canjs.md]: It does not work, `StacheElement` is not attached to the `can` object.

Comment (phillipskevin) [reply docs/can-guides/setup/setting-up-canjs.md]: I'm sure it's not working. We'll need to add can-namespace to the new packages and add these to it. We can merge this PR for now and fix that after.

**Pull request merged**